src/components/magnet/gradient-field.tsx

```diff
@@ -15,15 +17,13 @@ export default PixiComponent<IProps, PIXI.Graphics>("Gradient", {
     return new PIXI.Graphics();
   },
   applyProps: (instance, oldProps, newProps) => {
-    if (JSON.stringify(oldProps) === JSON.stringify(newProps)) return;
-
-    const { magnets, width, height, cellSize } = newProps;
+    const { magnets, magnetModels, width, height, cellSize } = newProps;
```

Comment:
@sfentress This is probably the change I was most concerned about - this JSON string check was preventing updates and I couldn't see a need for it so I got rid of it. Do you foresee any issues with that?